Automattic/wp-calypso
# Label premium themes & v-themes as Upgrade

Related to https://github.com/Automattic/dotcom-forge/issues/1654

## Proposed Changes

Premium themes and virtual themes made of style variations of premium and non-premium themes require the Premium or above plan to use. These themes and v-themes are labelled as Premium. This change labels them as 'Upgrade' instead.

## Testing Instructions



1. Use calypso.live link from the comment below
2. Go to http://calypso.localhost:3000/setup/site-setup/designSetup?siteSlug=YOURFREESITE.wordpress.com&flags=virtual-themes/onboarding
3. See how the premium badges all say "Upgrade"
4. Reload the page using a premium site
5. See how the premium badges for premium themes all say "Premium" rather than "Upgrade"
6. See how the style variations don't mention premium at all (maintains existing behaviour)

## Pre-merge Checklist
